lib/filesystem/include/filesystem/filesystem.hpp

```diff
@@ -142,6 +145,47 @@ namespace irods::experimental::filesystem
 
         auto permissions(rxComm& _comm, const path& _p, const std::string& _user_or_group, perms _prms) -> void;
 
+        /// \brief Modifies the permissions of a collection or data object.
+        ///
+        /// \throws filesystem_error If the path is empty, exceeds the path limit, or does not
+        ///                          reference a collection or data object.
+        ///
+        /// \param[in] _admin         A tag which instructs the function to operate in administrator mode.
+        ///                           The client must be an administrator to use this function.
+        /// \param[in] _comm          The communication object.
+        /// \param[in] _p             The logical path to a collection or data object.
+        /// \param[in] _user_or_group The user or group for which the permission will apply to.
+        /// \param[in] _prms          The permission to set for \p _user_or_group.
+        ///
+        /// \since 4.2.11
+        auto permissions(admin_tag, rxComm& _comm, const path& _p, const std::string& _user_or_group, perms _prms) -> void;
+
+        /// \brief Modifies the inheritance option of a collection.
+        ///
+        /// \throws filesystem_error If the path is empty, exceeds the path limit, or does not
+        ///                          reference a collection.
+        ///
+        /// \param[in] _comm  The communication object.
+        /// \param[in] _p     The path to a collection.
+        /// \param[in] _value A boolean indicating whether inheritance should be enabled or not.
+        ///
+        /// \since 4.2.11
+        auto enable_inheritance(rxComm& _comm, const path& _p, bool _value) -> void;
+
+        /// \brief Modifies the inheritance option of a collection.
+        ///
+        /// \throws filesystem_error If the path is empty, exceeds the path limit, or does not
+        ///                          reference a collection.
+        ///
+        /// \param[in] _admin A tag which instructs the function to operate in administrator mode. The client
+        ///                   must be an administrator to use this function.
+        /// \param[in] _comm  The communication object.
+        /// \param[in] _p     The path to a collection.
+        /// \param[in] _value A boolean indicating whether inheritance should be enabled or not.
+        ///
+        /// \since 4.2.11
+        auto enable_inheritance(admin_tag _admin, rxComm& _comm, const path& _p, bool _value) -> void;
```

Comment:
was it intentional to put the admin flag first?    rather than last?   then the two signatures (admin and non-admin) would be more similar?

first is definitely more 'visible'.

Reply:
Yes, it was intentional. The admin flag being first makes it so that no one can miss it. That isn't the only reason, but it is a big one.

Reply:
sold.